egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -43,11 +44,28 @@ def GetArgs():
     num_target_group.add_argument("--tree-dir", type=str,
                                   help="directory with final.mdl, from which we derive the num-targets")
 
+    # CNN options
+    parser.add_argument('--cnn.layer', type=str, action='append', dest = "cnn_layer",
+                        help="CNN parameters at each CNN layer, e.g. --filt-x-dim=3 --filt-y-dim=8 "
+                        "--filt-x-step=1 --filt-y-step=1 --num-filters=256 --pool-x-size=1 --pool-y-size=3 "
+                        "--pool-z-size=1 --pool-x-step=1 --pool-y-step=3 --pool-z-step=1, " 
+                        "when CNN layers are used, no LDA will be added", default = None)
+    parser.add_argument("--cnn.bottleneck-dim", type=int, dest = "cnn_bottleneck_dim",
+                        help="Output dimension of the linear layer at the CNN output "
+                        "for dimension reduction, e.g. 256."
+                        "The default zero means this layer is not needed.", default=0)
+    parser.add_argument("--cnn.cepstral-lifter", type=float, dest = "cepstral_lifter",
+                        help="Here we need the scaling factor on cepstra in the production of MFCC"
+                        "to cancel out the effect of lifter, e.g. 22.0", default=22.0)
+
     # General neural network options
     parser.add_argument("--splice-indexes", type=str, required = True,
-                        help="Splice indexes at each layer, e.g. '-3,-2,-1,0,1,2,3'")
+                        help="Splice indexes at each layer, e.g. '-3,-2,-1,0,1,2,3' "
+                        "If CNN layers are used, the first frame indices is actually how we "
+                        "splice the frames and pass to the CNN.")
     parser.add_argument("--add-lda", type=str, action=nnet3_train_lib.StrToBoolAction,
-                        help="add lda layer", default=True, choices = ["false", "true"])
+                        help="add lda layer (this option will be ignored when CNN layers are used", 
```

Comment:
<code> If "true" an LDA matrix computed from the input features (spliced according to the first set of splice-indexes) will be used as the first Affine layer. This affine layer's parameters are fixed during training. If --cnn.layer is specified this option will be forced to "false".  </code>